readme.markdown

```diff
@@ -90,7 +90,7 @@ var watchify = require('watchify');
 
 ## watchify(b, opts)
 
-watchify is a browserify [plugin](https://github.com/substack/node-browserify#bpluginplugin-opts), so it can be applied like any other plugin.
+watchify is a browserify [plugin](https://github.com/browserify/node-browserify#bpluginplugin-opts), so it can be applied like any other plugin.
```

Comment:
This URL should be for browserify/browserify, without the node- prefix, because the repository was also renamed
```suggestion
watchify is a browserify [plugin](https://github.com/browserify/browserify#bpluginplugin-opts), so it can be applied like any other plugin.
```